src/config.js

```diff
@@ -3,43 +3,29 @@ const TITLE_FILE = '_title.md';
 const SUMMARY_FILE = 'summary.md';
 const IMAGE_EXTENSIONS = ['.png'];
 const MD_EXTENSION = '.md';
-const HTML_EXTENSION = '.html';
 const MARKDOWN_EXTENSIONS = [MD_EXTENSION];
-const HTML_DIRECTORY = 'html';
 const MARKDOWN_DIRECTORY = 'markdown';
 const MARKDOWN_DIRECTORY_FOR_PDF = 'markdown_for_pdf';
 const PDF_DIRECTORY = 'pdf';
 const DIRECTORY = 'directory';
 const PDF_FILE = 'output.pdf';
 const STYLES_DIRECTORY = 'styles';
 const PDF_STYLES_FILE = 'pdf-styles.css';
-const HTML_STYLES_FILE = 'html-styles.css';
 // Markdown supports a maximum of 6 nesting levels
 const MAX_SUPPORTED_SECTION_LEVELS = 6;
 const SECTION_LEVELS = 6;
 const INTRO_FILE = '_intro.md';
 const PAGE_BREAK = '<div class="page-break"></div>';
 const SUMMARY_FILE_TITLE = '# Table of Contents';
```

Comment:
This will have to be changed if we want the Table of Contents to be at a smaller font. For now, using #